package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ui-utils",
-  "version": "2.3.2",
+  "version": "2.4.0",
```

Comment:
This needs to be 2.4.3, since that's the next version for ui-admin, so we match these versions, Angular style. Could also please make sure that the latest (you can do that after you merge this) commit is git-tagged as 2.4.3 as well.

Reply:
I made a commit now using `npm version`, which bumped automatically the `package.json` and `package-lock.json`. It also created the tag.

<img width="687" alt="image" src="https://github.com/rosebud-iot/ui-utils/assets/105205711/bb6dc1f8-cef4-4684-a374-4167c9aa8824">